### ISSUE TITLE
autoid and accented characters not properly rendered

### ISSUE THREAD

**Issue by bflorat** [asciidoctor/asciidoctor-vscode#281]
## Description
Links to sections with autoid and accented character doesn't work.

Thanks for this great plugin, very useful !

## System Information

Asciidoc 2.7.13

VSCode :
Version: 1.43.1
Commit: fe22a9645b44368865c0ba92e2fb881ff1afce94
Date: 2020-03-18T07:17:38.324Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.0.0-38-generic snap

## To Reproduce
```
=== target accented é
=== target non accented
[[target]]
=== target accented é 2

=== chapitre 2


see <<target accented é>>
see <<target non accented>>
see <<target>>
```

Anchor `<<target non accented>>` fine
Anchor  ` accented é>>` has no effect
Anchor  `<<target>>` works fine

* No problem with HTML5 or PDF export.

**Comment by danyill** [comment]
@bflorat thanks for filing this issue and for your example document. When I save the asciidoc to html it works fine, so I can confirm the issue:

The `h3` element looks well formed:

```html
<h3 id="_target_accented_é">target accented é</h3>
```

as does the anchor:

```html
<a href="#_target_accented_é" title="#_target_accented_é">target accented é</a>
```
So the Asciidoctor processing is working fine. I don't presently see where the issue arises but there's definitely something wrong.

**Comment by danyill** [comment]
I have a local fix for this -- details and a PR before too long. Currently trying to tidy up a few other "technical debt" things in the extension associated with the build process.

**Comment by bflorat** [comment]
Fine, thanks a lot.
I'll test it as soon as you release it.

**Comment by bflorat** [comment]
Thanks for the fix but I cannot see any difference. I can still reproduce the problem with the sample provided bellow.

I'm available for any further test if needed.

Plugin version 2.7.15
All default settings.
asciidoctor -v
Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.7.0p0 (2019-12-25 revision 647ee6f091) [x86_64-linux-gnu]) (lc:UTF-8 fs:UTF-8 in:UTF-8 ex:UTF-8)


VSCode : Version: 1.44.2
Commit: ff915844119ce9485abfe8aa9076ec76b5300ddd
Date: 2020-04-16T17:50:03.709Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.4.0-28-generic

**Comment by bflorat** [comment]
I confirm this is fixed now (tested successfully in 2.8.3)
